pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1223,13 +1252,17 @@ func TestNodeOperators(t *testing.T) {
 				t.Error(err)
 			}
 			newNode, found := cachedNodes.nodeInfoMap[node.Name]
-			if !found || len(cachedNodes.nodeInfoMap) != 1 {
+			if !found || len(cachedNodes.nodeInfoMap) != len(tc.nodes) {
 				t.Errorf("failed to dump cached nodes:\n got: %v \nexpected: %v", cachedNodes, cache.nodes)
```

Comment:
pls update all those error msg, got is actually `cachedNodes.nodeInfoMap` and expected is `tc.nodes`